src/components/Modal/ModalContainer.tsx

```diff
@@ -1,37 +1,50 @@
 import React, { forwardRef } from 'react'
 
-import { useLocale } from '../../i18n'
 import { ExternalStyles, Theme, useStyles } from '../../styles'
 import { Omit } from '../../util'
-import { Button } from '../Button'
-import { Icon } from '../Icon'
-import { Tooltip } from '../Tooltip'
+import { ModalCloseButton } from './ModalCloseButton'
+import { HeaderIconType, ModalHeader } from './ModalHeader'
 
-export interface ModalContainerProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'style'> {
+export interface ModalContainerProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'style' | 'title'> {
   style?: ExternalStyles
   hasCloseIcon?: boolean
   onClose?(): any
+  title?: string
+  subtitle?: string
+  headerIcon?: HeaderIconType
+  headerBackground?: string
+  showHeaderBorder?: boolean
```

Comment:
```suggestion
 header?: {
  icon?: HeaderIconType
  background?: string
  showBorder?: boolean
 }
```
I think it'll be better organized this way